src/sentry/api/endpoints/organization_events_root_cause_analysis.py

```diff
@@ -109,6 +114,80 @@ def query_spans(transaction, regression_breakpoint, params):
     return span_results
 
 
+def fetch_span_analysis_results(transaction_name, regression_breakpoint, params, project_id):
+    span_data = query_spans(
+        transaction=transaction_name,
+        regression_breakpoint=regression_breakpoint,
+        params=params,
+    )
+
+    span_analysis_results = span_analysis(span_data)
+
+    for result in span_analysis_results:
+        result["span_description"] = get_span_description(
+            EventID(project_id, result["sample_event_id"]),
+            result["span_op"],
+            result["span_group"],
+        )
+
+    return span_analysis_results
+
+
+def fetch_geo_analysis_results(transaction_name, regression_breakpoint, params, project_id):
+    def get_geo_data(period):
+        # Copy the params so we aren't modifying the base params each time
+        adjusted_params = {**params}
+
+        if period == "before":
+            adjusted_params["end"] = regression_breakpoint - BUFFER
+        else:
+            adjusted_params["start"] = regression_breakpoint + BUFFER
+
+        geo_code_durations = metrics_query(
+            ["p95(transaction.duration)", "geo.country_code", "count()"],
+            f"event.type:transaction transaction:{transaction_name} project_id:{project_id}",
```

Comment:
I think project id is passed in through adjusted_params so don't need to add it to the query